readthedocs-theme/templates/index.html

```diff
@@ -0,0 +1,113 @@
+{% extends "base.html" %}
+{# This is the blog index, not the home page template #}
+
+
+{% block content %}
+
+  <!-- Intro -->
+  <section class="ui vertical segment" id="blog-intro">
+    <div class="ui container">
+      <div class="ui segment basic">
+
+        {% block content_title %}
+          <div class="ui basic vertical big segment">
+          <h1 class="ui header">
+            <span class="ui header">Read the Docs Blog</span>
+          </h1>
+            <p>
+              Short intro
+            </p>
+          </div>
+        {% endblock %}
+
+      </div>
+    </div>
+  </section>
+
+
+  <section class="ui vertical segment" id="articles-list">
+    <div class="ui container">
+      <div class="ui segment basic">
+
+        <!-- Latest  2 articles -->
+        <div class="ui two cards stackable">
+          {% for article in articles_page.object_list %}
+            {% if not articles_previous_page and loop.index <= 2 %}
+              <a class="ui card" href="/{{ article.url }}">
+                {% if article.image %}
+                  <div class="image">
+                    <img src="{{ article.image }}">
+                  </div>
```

Comment:
This is another area where we should have blog posts with test images.

It is probably going to make me want images on all blog posts though, but i'm not sure how we make both types of posts work together on this page. EthicalAds forces images on all blog posts, which is definitely a direction we could go with the addition of a placeholder of sorts.

Reply:
I'm almost always in favor of adding more graphics; although instead of enforcing, I would probably add a default image instead, to be used when no other is provided.